page_menus.c

```diff
@@ -173,15 +173,38 @@ void delete(Book* k)
     {
     case 0:
     {
-        DB_delete(k->ISBN, BookT);
-        DB_delete(k->ISBN, BookRST);
+        ComparisonPair cp[] = {{compareByRentalBookISBN, k->ISBN}};
 
-        clearScreen();
-        printf("%s", getTranslation("del_suc", activeUser.language));
-        pressEnter();
+        int num_found, num_active = 0;
 
-        clearScreen();
-        mainPaige();
+        Rental* rentals = DB_select(RentalT, cp, sizeof(cp), &num_found);
+        for (int i = 0; i < num_found; i++)
+        {
+            if (rentals[i].ReturnDate.year == 0)
```

Comment:
Again did this. You could have just used the compare function to get only active ones. Leave it at this now